circle.yml

```diff
@@ -21,8 +21,10 @@ deployment:
     production:
         tag: /v[0-9]+(\.[0-9]+)*/
         commands:
-            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/docs/ --delete
+            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/$CIRCLE_TAG/docs/ --delete
+            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/latest/docs/ --delete
     develop:
         branch: master
         commands:
-            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/dev/docs/ --delete
+            - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/dev/$CIRCLE_TAG/docs/ --delete
```

Comment:
So:
`s3://sceptre.cloudreach.com/$CIRCLE_TAG/docs/` for tagged commits
`s3://sceptre.cloudreach.com/$CIRCLE_SHA1/docs/` for other commits to master

`s3://sceptre.cloudreach.com/latest/docs/` points to latest tagged commit - (i.e. hopefully latest stable version)
`s3://sceptre.cloudreach.com/dev/docs/` points to newest commit